lighthouse-cli/printer.js

```diff
@@ -113,7 +113,8 @@ function createOutput(results, outputMode) {
   // Pretty printed.
   const bold = '\x1b[1m';
   const reset = '\x1B[0m';
-  let output = `\n\n${bold}Lighthouse results:${reset} ${results.url}\n\n`;
+  const version = results.meta.lighthouseVersion;
+  let output = `\n\n${bold}Lighthouse (${version}) results:${reset} ${results.url}\n\n`;
```

Comment:
I might suggest switching this up to output:

```
Lighthouse results: ....
Version: ${version}
```

I would personally find it more readable for the CLI, but the HTML report otherwise is 💯